src/argus_htmx/tailwindtheme/snippets/20-loading-box.css

```diff
@@ -1,18 +1,25 @@
 @layer components {
+   /*
+    * Example usage:
+    * <div class="loading-box">
+    *   <!-- Any content tag here -->
+    *   <div class="htmx-indicator">
+    *     <!-- Any optional feedback widget here -->
+    *   </div>
+    * </div>
+    */
```

Comment:
css and html looks cleaner so that's nice. It does mean that you can only ever put two element inside  `loading-box` but I guess that's fine :)

Reply:
fair point 👍